bin/run_docker.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+"""
+This module runs the docker container and outputs relevant log messages into a log file
+which then gets stored on Synapse under its corresponding log folder.
+
+This module recycles functionality seen here:
+https://github.com/Sage-Bionetworks-Challenges/model-to-data-challenge-workflow/blob/82ff3dc0ea8b83d727a7fbecf9550efdc010eadd/run_docker.py
+"""
+
+import sys
+import os
+import docker
+import create_folders, get_submission_image
```

Comment:
minor nit: Could also:
```
from create_folders import create_folders
```

So you can directly call the `create_folders` function